app/views/split_checkout/_voucher_section.cable_ready.haml

```diff
@@ -0,0 +1,21 @@
+%div#voucher-section
+  = form_with url: checkout_update_path(:payment), model: order, method: :put, data: { remote: "true" } do |f|
```

Comment:
This partial is being rendered inside of another form, so creating a second form here won't work as intended (it's a weird rule with HTML and browsers that you can't ever have a form inside a form). I think different browsers (and versions) might even handle this differently and give different results.

So effectively the submit button here is just submitting the main form on the checkout/payment page (with all fields).